planetarium/NineChronicles
# Apply recipe entrance animation.

### Description

SSIA

### How to test

1. Check if recipe entrance animation is played properly.
2. Check if equipment recipe row's name is changed to non-colored.

### Required Reviewers

@Namyujeong @planetarium/9c-dev
